tft.py

```diff
@@ -401,12 +402,16 @@ def exit_now_conditional() -> bool:
     return not league_game_already_running()
 
 
-def check_if_game_complete() -> bool:
+def check_if_game_complete(wait_for_exit_buttons: bool = False) -> bool:
     """Check if the League game is complete.
 
     Returns:
         bool: True if any scenario in which the game is not active, False otherwise.
     """
+    if wait_for_exit_buttons:
+        logger.info("Waiting an additional ~25s for any exit buttons")

```

Comment:
Do we want to check during this time? At present this will just wait for 25 seconds.

Reply:
I've also been thinking about our classic "While True: Look for the button, increase a timeout, and if the timeout is reached, THEN try the failsafe" approach. I'll change it to that.